### ISSUE TITLE
No animation when open the recent apps screen from niagra. It just hard cuts to the recent apps screen. 

### ISSUE THREAD

**Issue by DaStoopidGuy** [8bitPit/Niagara-Issues#2707]

**Comment by NiagaraSupport** [comment]
Hey there, thanks for the report.
We don't interact much with animations. We utilize them if the system provides an API for us to use. If not, we can't create them. The problem might be related to your OS/hardware. Does the issue also happen on other third-party launchers?

**Comment by DaStoopidGuy** [comment]
Yes, It does happen on other third-party launchers, but only when theyre set to default and not just opened through the app. When for example, hyperion was opened through the app, it worked flawlessly when navigating to the recents screen. The home screen animated like any other app. But when i set it to default and opened recent apps, no animation. I'm on ColorOS 14 (oppo) btw.

**Comment by NiagaraSupport** [comment]
Checking through some older tickets right now. Has this issue been fixed, or is it still relevant?

**Comment by DaStoopidGuy** [comment]
Yeah it still happens but ive gotten used to it lol. Niagara is too nice to leave just because of a few minor issues :>

**Comment by NiagaraSupport** [comment]
Thanks for the reply, and glad you're still using the Launcher.
I'll see if there is something we can do about this issue.
